### PR TITLE
UI improves

### PR DESCRIPTION
**Pull request by drkmen**

### CODE REVIEW

File: app/views/employees/show.html.haml [hunk 1]
```diff
@@ -40,6 +40,16 @@
               %p
                 = @employee.description
 
+        - if @employee.additional.present?
+          %h3 Additional information:
+          .row
+            .col-12
+              - @employee.additional.each do |key, value|
+                %b= key.humanize + ':'
```
Comment (Tivgres) [inline app/views/employees/show.html.haml]: Maybe better to use 
```
%b="#{key.humanize}:#{value}"
```

File: app/views/employees/show.html.haml [hunk 2]
```diff
@@ -40,6 +40,16 @@
...
+                = value
+                %br
+
+        %br
```
Comment (Tivgres) [inline app/views/employees/show.html.haml]: Use %p instead twice %br

Comment (drkmen) [reply app/views/employees/show.html.haml]: they are at different levels, first `%br` do a new line after each field, second `%br` after the whole section

Comment (Tivgres) [reply app/views/employees/show.html.haml]: But we can use %p at line #52 and it's look like same, no?

File: app/views/employees/show.html.haml (hunk 1 above)
Comment (drkmen) [reply app/views/employees/show.html.haml]: No, because only `key` should be bold.

**Pull request merged**